### PR TITLE
Tasks51-60

### PR DESCRIPTION
**Pull request by nazar-ginailo**
New resolved tasks 51-60

### CODE REVIEW

File: tasks/54_roman-numbers-convert/roman-numbers-convert.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+export function convertToRoman(num) {
+  const bank = {M: 1000, CM: 900, D: 500, CD: 400, C: 100,
```
Comment (Tsvetkovv) [inline tasks/54_roman-numbers-convert/roman-numbers-convert.js]: `const ROMANIAN_NUMBERS`

File: tasks/57_pig-latin/pig-latin.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+export function translatePigLatin(str) {
+  const bank = 'iueoa';
```
Comment (Tsvetkovv) [inline tasks/57_pig-latin/pig-latin.js]: vowels?

File: tasks/54_roman-numbers-convert/roman-numbers-convert.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  let result = '';
+  for (let key in ROMANIAN_NUMBERS) {
+    if (ROMANIAN_NUMBERS.hasOwnProperty(key)) {
+      while (num >= ROMANIAN_NUMBERS[key]) {
+        result += key;
+        num -= ROMANIAN_NUMBERS[key];
+      }
+    }
+  }
+  return result;
```
Comment (ArtemLunt) [inline tasks/54_roman-numbers-convert/roman-numbers-convert.js]: Looks too complex.
Maybe smth lilke map?

File: tasks/57_pig-latin/pig-latin.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+export function translatePigLatin(str) {
+  const VOWELS = 'iueoa';
+  if (VOWELS.includes(str.charAt(0))) {
+    return str + 'way';
+  }
+  let index = 1;
+  while (!VOWELS.includes(str.charAt(index))) {
+    index++;
+  }
+  return str.slice(index) + str.slice(0, index) + 'ay';
```
Comment (ArtemLunt) [inline tasks/57_pig-latin/pig-latin.js]: Pls make breaks between the different logical parts

File: tasks/58_missed-letters/missed-letters.js [hunk 5]
```diff
@@ -0,0 +1,8 @@
+export function findLetter(str) {
+  for (let i = 0; i < str.length; i++) {
```
Comment (ArtemLunt) [inline tasks/58_missed-letters/missed-letters.js]: Foreach?

**Pull request merged**